master/getting-started/bare-metal/bare-metal.md

```diff
@@ -36,6 +36,9 @@ to/from other interfaces is left alone.
 > case, the host endpoint policy would need to be very broad to allow all
 > traffic destined for any possible workload.
 >
+> Since version 2.1.0, Calico applies host endpoint policy to traffic that is
+> being forwarded between host endpoints.
+>
```

Comment:
I think this is fine.  Perhaps one might want to tweak or add to cover the case of forwarding to another host _interface_ that is not configured as a host _endpoint_ - but unless there's a really snappy way of doing that, it feels like a case where less (as already proposed) might be more.